### PR TITLE
Asic

### PR DESCRIPTION
**Pull request by microSharjeel**
Hi Sir 
Some changes in README 
regards

### CODE REVIEW

File: hardware/asic/README.md [hunk 1]
```diff
@@ -327,7 +331,7 @@ As the skywater tech files are not installed in magic’s library, we need to cr
 
 So enter following for the symbolic links:
 ```bash
-sudo ln -s <sky130A_install_root_dir>/sky130A/libs.tech/magic/*../usr/local/lib/magic/sys/
+sudo ln -s <sky130A_install_root_dir>/sky130A/libs.tech/magic/    /usr/local/lib/magic/sys/
```
Comment (jjts) [inline hardware/asic/README.md:334]: I did not need this 

One of the things I installed automatically copied the files to /usr/local/lib/magic/sys/

You may have missed some "make install" step

Comment (microSharjeel) [reply hardware/asic/README.md:334]: Sir I followed the following approach
As the skywater tech files are not installed in magic’s library, we need to create a symbolic link in order to use the tech files for drawing layout.

Comment (jjts) [reply hardware/asic/README.md:334]: You are right but I think the * I needed 
sudo ln -s <sky130A_install_root_dir>/sky130A/libs.tech/magic/*   /usr/local/lib/magic/sys/

Comment (microSharjeel) [reply hardware/asic/README.md:334]: ok I will add in the next push plz merge this and i will push another PR also with PDK_ROOT removed

**Comment by microSharjeel** [comment]
Now all changes are there in README as discussed yesterday
regards

**Pull request merged**